src/main/java/com/linkedin/kmf/services/SignalFxMetricsReporterService.java

```diff
@@ -0,0 +1,197 @@
+/**
+ * Copyright 2016 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
```

Comment:
That's not correct. I don't think LinkedIn would own the copyright for this file. Check with the contribution guidelines of the project.

Reply:
Yes, I tried to change it, but it has checkstyle config which checks for this config 
https://github.com/signalfx/kafka-monitor/blob/master/checkstyle/checkstyle.xml#L21. What's your thought on this?

Reply:
You should check what their contributor agreement is, and whether they require that they own the copyright, etc.

Reply:
@mpetazzoni @ramjothikumar thinks we should involve legal team here.